### PR TITLE
Clarify how to contribute

### PR DESCRIPTION
**Pull request by saad-ali**
To promote a culture of transparency and inclusion, I'm documenting how to get involved in the project, and documenting the role of the approvers group.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,51 @@
...
+The Container Storage Interface (CSI) Community aims to be open, transparent, inclusive, and promote the best ideas.
+Contributions are welcome and encouraged from everyone.
 
-We hold a public meeting every two weeks, on Wednesdays at 9 AM (PT).
+### Github Org
 
-The meeting is held on Zoom: [meeting link](https://zoom.us/j/790748945).
-Agenda and any notes from each meeting are captured in [this document](https://docs.google.com/document/d/1-oiNg5V_GtS_JBAEViVBhZ3BYVFlbSz70hreyaD7c5Y/edit).
-Join container-storage-interface-community@googlegroups.com or email saadali@google.com for a calendar invite.
+If there are issues or chanages you would like to see, feel free to open an issue or send out a PR in the approprite repo (e.g. https://github.com/container-storage-interface/spec/issues).
```
Comment (jdef) [inline README.md]: spelling: appropriate

Comment (jdef) [reply README.md]: also: chanages

File: README.md [hunk 2]
```diff
@@ -1,15 +1,51 @@
...
+Review and approval process is defined in the [governance doc](https://github.com/container-storage-interface/community/blob/master/governance.md).
```
Comment (jdef) [inline README.md]: s/Review/The review/ ?

File: README.md [hunk 3]
```diff
@@ -1,15 +1,51 @@
...
+
+### Community Google Group
+CSI community announcements, event invites, and discussions are posted to the [container-storage-interface-community](https://groups.google.com/forum/#!forum/container-storage-interface-community) Google Group.
+Join the group to participate.
+
+## Approvers
+
+The [governance doc](https://github.com/container-storage-interface/community/blob/master/governance.md) defines the role of approvers.
+
+Becuase communities attempting to define industry standards are susceptible to conflict between vendors or becoming primarily focused on a single vendor. The governance doc for this project limits membership of approvers to representatives of the COs.
```
Comment (jdef) [inline README.md]: spelling: Becuase.  Maybe just remove the word entirely so that the sentence reads better?

Comment (jdef) [reply README.md]: also, should the second sentence start on a new line?

File: README.md [hunk 4]
```diff
@@ -1,15 +1,51 @@
...
+In order to be successful, CSI must remain vendor neutral and serve the needs of end users.
+
+Ideas, however, should be generated, discussed, debated, and consensus achieved publicly within the community.
+Approvers should only have to step in or "overrule" when absolutely necessary (a vendor is clearly pushing a singularly beneficial agenda, two storage vendors strongly disagree, etc.).
+
+### Approvers Group Meetings
+
+A CSI Approvers Group meeting is held every two weeks. Attendence is limited to the approvers as defined in the [governance doc](https://github.com/container-storage-interface/community/blob/master/governance.md).
```
Comment (jdef) [inline README.md]: separate lines for different sentences

**Review by jdef — COMMENTED** [review]
Thanks for this, @saad-ali .

Completed a first review pass, commented on several typos and nits.

**Comment by saad-ali** [comment]
@jdef PTAL

**Review by jdef — APPROVED** [review]
LGTM

**Pull request merged**